### ISSUE TITLE
[cmake 3.12.3] libuv-1.dll finding issue for cmake-gui

### ISSUE THREAD

**Issue by jannick0** [msys2/MINGW-packages#4583]
`cmake-gui` (version 3.12.3) does not make it to get fired up due to a finding issue with `libuv-1.dll`:
~~~
MINGW64 /mingw64
$ cmake --version
cmake version 3.12.3

MINGW64 /mingw64
$ cmake-gui
C:/msys64/mingw64/bin/cmake-gui.exe: error while loading shared libraries: libuv-1.dll: cannot open shared object file: No such file or directory
~~~
... whereas `libuv-1.dll` **is** installed:
~~~
MINGW64 /mingw64/bin
$ ls libuv*.dll
libuv-1.dll
~~~
... and funnily from `bin` the gui does not find the dll, either. Issue as to how the path to the dll is constructed?
~~~
MINGW64 /mingw64/bin
$ cmake-gui
C:/msys64/mingw64/bin/cmake-gui.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
~~~
Many thanks in any event!

**Comment by revelator** [comment]
problem with exports maybe ?.
not the first time i seen that, but just in case try rebuilding libuv and cmake in that order and see what happens.

**Comment by jannick0** [comment]
Thanks for the tip to rebuild!  It took me some time (not all of the two months delay ;) ) to look how the `makepkg` machinery works. Then I did a build ride ... and now cmake-gui opens without complaining.

Q - Would it be possible that the `optdepends` as defined in cmake's `PKGBUILD` is required for `cmake-gui`?
https://github.com/Alexpux/MINGW-packages/blob/c4ac19f8f43596168cf293c5c35824e7b5089073/mingw-w64-cmake/PKGBUILD#L24
If so, here possible solutions:
* add `qt5` to the runtime `depends` or
* split packages `cmake` and `cmake-gui`?
The first is easy, but disk space consuming, the second I do not know how to that exactly.

**Comment by jannick0** [comment]
Solved in `cmake 3.15.0` or earlier.
